test/backend_test.in.cpp

```diff
@@ -4764,6 +4766,30 @@ NGRAPH_TEST(${BACKEND_NAME}, reverse_sequence_n4d2c3h2w2)
     EXPECT_EQ(read_vector<int>(result), expected);
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, generate_mask)
+{
+    Shape scalar{};
+    Shape result_shape{1, 20};
```

Comment:
I'm being a dork here but can we increase this to, say, `{1, 128}`? With `{1, 20}` (and if the seed weren't fixed), the probability of one of the `ASSERT_NE` tests below failing on a compliant implementation is only about 1/500000  :)

Reply:
I should have said: the probability is about 1/500000, which is a bit high :)